### DISCUSSION TITLE
[PSD2] Exercise 1.64

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#128] [Esercizi (A-L)]
**Exercise 1.64** In a binary coded decimal (BCD) system, 4 bits are used to represent a decimal digit from 0 to 9. For example, $37_{10}$ is written as $0011\\\_0111_{BCD}$.

(a) Write $289_{10}$ in BCD
(b) Convert $1001\\\_0101\\\_0001_{BCD}$ to decimal
(c) Convert $0110\\\_1001_{BCD}$ to binary
(d) Explain why BCD might be a useful way to represent numbers

**Comment by Elia-Belli** [comment]
- (a) $289_{10} = 0010 \\_ 1000\\_ 1001_{BCD}$
- (b) $1001\\_ 0101\\_ 0001_{BCD} = 951_{10}$
- (c) $0110\\_ 1001_{BCD} = 69_{10} = 1000101_2$
- (d) La codifica BCD è più leggibile dall' uomo, anche per grandi numeri è possibile leggere direttamente le varie cifre una ad una.

**Comment by matypist** [comment]
> **Errore di Formattazione**: nel testo per 3710 e 28910 si intende 3710 e 28910 , infatti 3710→0011_0111 in BCD.

Errore di formattazione corretto, grazie per la segnalazione :-)
